charts/model-engine/templates/restart_keda_operator.yaml

```diff
@@ -0,0 +1,54 @@
+{{- if .Values.restartKedaOperator }}
```

Comment:
is there a comment somewhere why we need this? to capture why we are putting this here in llm engine?